data/omim.tsv

```diff
@@ -1,2 +1,1375 @@
 #did	when	nextofkin
 309150	2011-06-10	614279
+100500	30/07/2025
```

Comment:
the 'when' is when OMIM deprecated the identifier, not when you added it. Leave it empty if it is not clear when they removed it.

Btw, the format is the following the ISO standard, so YYYY-MM-DD